docs/docs/setting-up/jobs/job-types.md

```diff
@@ -1,40 +1,40 @@
-description: The different job types available in Bacalhau
+description: Job types in Bacalhau
 ---
 
-Bacalhau has recently introduced different job types in v1.1,
+Bacalhau supports different job types,
 providing more control and flexibility over the orchestration and scheduling of those jobs - depending on their type.
 
-Despite the differences in job types, all jobs benefit from core functionalities provided by Bacalhau, including:```
+Despite the differences in job types, all jobs benefit from core functionalities provided by Bacalhau, including:
 
 1. **Node selection** - the appropriate nodes are selected based on several criteria, including resource availability, priority and feedback from the nodes.
-2. **Job monitoring** - jobs are monitored to ensure they complete, and that they stay in a healthy state.  
+2. **Job monitoring** - jobs are monitored to ensure they complete, and that they stay in a healthy state.
 3. **Retries** - within limits, Bacalhau will retry certain jobs a set number of times should it fail to complete successfully when requested.
 
 
-## 1 Batch Jobs
+## Batch Jobs
 
 Batch jobs are executed on demand, running on a specified number of Bacalhau nodes. These jobs either run until completion or until they reach a timeout. They are designed to carry out a single, discrete task before finishing.
 
-Ideal for intermittent yet intensive data dives, for instance performing computation over large datasets before publishing the response.  This approach eliminates the continuous processing overhead, focusing on specific, in-depth investigations and computation.
+Ideal for intermittent yet intensive data dives, for instance performing computation over large datasets before publishing the response. This approach eliminates the continuous processing overhead, focusing on specific, in-depth investigations and computation.
 
 ## Ops Jobs
 
-Similar to batch jobs, ops jobs have a broader reach. They are executed on all nodes that align with the job specification, but otherwise behave like batch jobs.
+Ops jobs are executed on all available nodes that match the requirements of the job. In all other aspects, they behave like Batch jobs.
 
-Ops jobs are perfect for urgent investigations, granting direct access to logs on host machines, where previously you may have had to wait for the logs to arrive at a central locartion before being able to query them. They can also be used for delivering configuration files for other systems should you wish to deploy an update to many machines at once. 
+Ops jobs allow to control the whole fleet, aggregate data from every avaiable node. They are perfect for urgent investigations, granting direct access to logs on host machines, where previously you may have had to wait for the logs to arrive at a central locartion before being able to query them. They can also be used for delivering configuration files for other systems should you wish to deploy an update to many machines at once. 
```

Comment:
> Ops jobs allow to control the whole fleet, aggregate data from every avaiable node.

would expect 

`
Ops jobs allow SOMEONE/SOMETHING to control the whole fleet.
`

> aggregate data from every avaiable node

I'm not sure this makes sense.